config/initializers/create_background_jobs.rb

```diff
@@ -0,0 +1,5 @@
+# Create all Rufus Scheduler Jobs on Application Start
```

Comment:
Just to be more precise ;)
```suggestion
# Create all Rufus Scheduler Jobs for active checks on Application Start
```
Thanks for your contribution!

Reply:
done